ui/views.py

```diff
@@ -240,7 +241,12 @@ def extra_context(self):
 @login_required
 def export(request, resource_id):
     """Dump LearningResource as XML"""
-    return HttpResponse(
-        get_resource(resource_id, request.user.id).content_xml,
-        content_type='text/xml'
-    )
+    try:
+        return HttpResponse(
+            get_resource(resource_id, request.user.id).content_xml,
+            content_type='text/xml'
+        )
+    except NotFound:
+        raise Http404()
+    except PermissionDenied:
+        return HttpResponseForbidden("unauthorized")
```

Comment:
That's fine, except I'd have to import `PermissionDenied` from both the API and Django, so I'd have to rename one. I don't mind doing that. Is it objectively better to raise Django's `PermissionDenied` here?